sunpy/net/dataretriever/sources/goes.py

```diff
@@ -98,6 +98,10 @@ def post_search_hook(self, i, matchdict):
         rowdict["Physobs"] = matchdict["Physobs"][0]
         rowdict["url"] = i["url"]
         rowdict["Source"] = matchdict["Source"][0]
+        if "avg1m" in i["url"]:
+            rowdict["Resolution"] = "avg1m"
+        else:
+            rowdict["Resolution"] = "flx1s"
```

Comment:
```suggestion
        if "avg1m" in i["url"]:
            rowdict["Resolution"] = "avg1m"
        elif "fix1s" in i["url"]:
            rowdict["Resolution"] = "flx1s"
        else:
            raise RuntimeError("Could not parse resolution from URL")
```

Reply:
To be on the safe side, and avoid putting an incorrect resolution in the return.